config.default.yaml

```diff
@@ -26,7 +26,7 @@ clustering:
 snapshots:
   start: "2013-01-01"
   end: "2014-01-01"
-  closed: 'left' # end is not inclusive
+  inclusive: 'left' # end is not inclusive
```

Comment:
This will break compatibility with quite recent pandas versions. We'll do it once needed with #323

Reply:
ah, haven't seen this PR, sorry